app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,478 @@
+<resources>
+    <string name="app_name" translatable="false">FastHub</string>
+    <string name="app_name_full" translatable="false">FastHub for Android</string>
```

Comment:
@caiorrs delete them if you pull the latest we have sorted them to be all on the top.

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_